tests/conftest.py

```diff
@@ -3236,21 +3236,40 @@ def generate_ascending_list_of_datetimes(
 
 
 @pytest.fixture
-def data_context_with_simple_sql_datasource_for_testing_get_batch(
-    sa, empty_data_context
-):
-    context = empty_data_context
-
+def sqlite_connection_string():
```

Comment:
It looks like the previous fixture didn't annotate the return type. I'm not too worried about it but it might be good to do.

Reply:
We should start type-checking more of our test modules.
The root `tests/conftest.py` would be a good place to start since so much relies on it.

Reply:
yes please - let's treat test code like production code.

I'd humbly request annotations on all fixture definitions and usages